astropy/io/ascii/tests/test_ecsv.py

```diff
@@ -292,6 +292,7 @@ def test_regression_5604():
 
 
 def assert_objects_equal(obj1, obj2, attrs, compare_class=True):
+    __tracebackhide__ = True
```

Comment:
Here I am much less sure, since really this is a combination of tests; I think in debugging it would actually be useful to know which part failed.

Reply:
good thinking, I'll clean it up.